samples/export.ts

```diff
@@ -0,0 +1,28 @@
+export const numberRegexp : string;
```

Comment:
Doesn't the TypeScript coding style recommend that there be no space before `:`?

Reply:
Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/54/commits/b661ef794dd6b4c726e9856f27e6c8d3a9b4694c